requires-testing.txt

```diff
@@ -6,6 +6,7 @@ pytest-mock==2.0.0;python_version=="2.7"
 lxml==4.5.0
 selenium==3.141.0
 percy==2.0.2
+cryptography==3.0
```

Comment:
Trying to fix https://app.circleci.com/pipelines/github/plotly/dash/1693/workflows/002acae1-f322-4389-928c-f2c14deaab1b/jobs/27184

Seems like there's something weird going on in the latest release (tonight!) of `cryptography`, which is pulled in by `requests` - see eg https://github.com/pyca/cryptography/pull/5412